### PR TITLE
IL 298 get avg rating

### PR DESCRIPTION
**Pull request by M437A**

### CODE REVIEW

**Review by annstriganova — APPROVED** [review]
Approved

File: src/main/resources/api-specs/rating-openapi.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ tags:
     description: "Operations related to rating products"
 
 paths:
-  /api/v1/products/rating:
+  /api/v1/rating/product:
```
Comment (Sunagatov) [inline src/main/resources/api-specs/rating-openapi.yaml:22]: Please, make it like this:
```
HTTP POST api/v1/products/{productId}/rating
```

File: src/main/resources/api-specs/rating-openapi.yaml [hunk 2]
```diff
@@ -44,6 +44,30 @@ paths:
               schema:
                 $ref: "#/components/schemas/RatingDto"
 
+  /api/v1/rating/product/{productId}:
```
Comment (Sunagatov) [inline src/main/resources/api-specs/rating-openapi.yaml:47]: Please, make it like this:
```
HTTP POST api/v1/products/{productId}/rating
```

**Review by Sunagatov — COMMENTED** [review]
Reviewed

**Pull request merged**